src/main/java/it/pn/frontend/e2e/common/RecapitiDestinatarioPage.java

```diff
@@ -358,9 +364,14 @@ public void clickSuConfermaElimina() {
         this.element(confermaRimuoviPECBy).click();
     }
 
-    public boolean siControllaNonPresenzaPEC() {
+    public void clickSuAnnullaElimina() {
+        this.getWebDriverWait(10).withMessage("Non è stato possibile cliccare sul bottone annulla").until(ExpectedConditions.elementToBeClickable(buttonAnnullaEliminazioneInPopUp));
+        buttonAnnullaEliminazioneInPopUp.click();
+    }
```

Comment:
Already exist: 
https://github.com/pagopa/pn-frontend-e2e/blob/10524309fbd27d6c7d78103d86d876080fe07053/src/main/java/it/pn/frontend/e2e/common/RecapitiDestinatarioPage.java#L564-L566

Just add the same check on the existent one